docs/adrs/0011-use-feature-managment-to-control-release-of-upcoming-features.md

```diff
@@ -0,0 +1,19 @@
+# 11. Use feature management to control the release of upcoming features
+
+Date: 2024-08-07
+
+## Status
+
+Accepted
+
+## Context
+
+Development of future features may have to be started before supporting work is completed. We need a system that allows us to create new features without having them released to users.
+
+## Decision
+
+Using the ASP.Net Feature managment nuget package we will implement feature flags. This will allow us to place upcoming features/pages behind feature gates meaning that development can be completed on future features and the release of these can be controlled using environment variables and using the app settings files.
+
```

Comment:
Should we also add the decision to keep a record of what each flag does in the docs and the decision to remove flags from the codebase as soon as they are done with?